helm_deploy/hmpps-book-secure-move-frontend/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+Application is running at:
```

Comment:
What is this file used for?

Reply:
Not much in this case, feel free to get rid of it.  It can be used to display a bunch of info about a successful deployment.  https://helm.sh/docs/chart_template_guide/notes_files/

Reply:
Removed.